cluster/manifests/node-monitor/service.yaml

```diff
@@ -1,20 +0,0 @@
-kind: Service
```

Comment:
Is this service to disappear? Shouldn't you apply similar selector / labels as in the other services?

Reply:
Forgot to mention why this is to be removed. It was never really used for anything so it can just be dropped. I added this to the description on top now.

Reply:
this broke monitoring to gather data prometheus -> node-exporter and cadvisor